src/ORM/DataObject.php

```diff
@@ -2454,6 +2454,17 @@ protected function beforeUpdateCMSFields($callback)
         $this->beforeExtending('updateCMSFields', $callback);
     }
 
+    /**
+     * Allows user code to hook into DataObject::getCMSFields after updateCMSFields
+     * being called on extensions
+     *
+     * @param callable $callback The callback to execute
+     */
+    protected function afterUpdateCMSFields($callback)
```

Comment:
Should we put a `callable` type hint here?

Reply:
I was following the signature for `beforeUpdateCMSFields` which has gone a long time without a type hint. If we add a type hint here we should also add one to that method.

Reply:
@sminnee What are your thoughts? Should I add a typehint to both methods? Or leave as is?